manifests/server/installdb.pp

```diff
@@ -19,7 +19,7 @@
       creates   => "${datadir}/mysql",
       logoutput => on_failure,
       path      => '/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin',
-      require   => Package[$mysql::server::package_name],
+      require   => Package['mysql-server'],
```

Comment:
This will still fail if `$mysql::server::package_manage` is false, since the package resource won't be in the catalog.